### PR TITLE
GG-24928 Removed Web Console from 8.8-master.

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -131,9 +130,8 @@
         </profile>
 
         <profile>
-            <id>all-other</id> <!-- used to update project versions and check all modules compilation -->
-            <modules> <!-- sorted alphabetically -->
-                <module>modules/web-console</module>
+            <id>all-other</id> <!-- Used to update project versions and check all modules compilation. -->
```
Comment (vveider) [inline pom.xml]: How about to remove this profile completely if no modules left there?

Comment (akuznetsov-gridgain) [reply pom.xml]: @vveider From my opinion, this is out of this issue scope.

Comment (vveider) [reply pom.xml]: But empty profile is a bug, leftover.
@nva, WDYT?

Comment (akuznetsov-gridgain) [reply pom.xml]: @vveider I removed this profile and its usages (DEVNOTES.txt, update-versions.sh, and build.sh in ggprivate repository).

**Pull request merged**